android/src/main/java/com/airbnb/android/react/maps/AirMapView.java

```diff
@@ -578,13 +578,16 @@ public boolean dispatchTouchEvent(MotionEvent ev) {
 
         switch (action) {
             case (MotionEvent.ACTION_DOWN):
-                this.getParent().requestDisallowInterceptTouchEvent(true);
+                if (map != null && map.getUiSettings().isScrollGesturesEnabled()) {
+                    this.getParent().requestDisallowInterceptTouchEvent(true);
```

Comment:
quick update: this pr definitely hasn't been forgotten. The current situation is causing a bug in our app, but I haven't had time to loop back to it yet.